runtime/jvmti/jvmtiStackFrame.cpp

```diff
@@ -548,9 +549,9 @@ jvmtiGetFrameLocation(jvmtiEnv *env,
 		if (rc == JVMTI_ERROR_NONE) {
 			j9object_t threadObject = (NULL == thread) ? currentThread->threadObject : J9_JNI_UNWRAP_REFERENCE(thread);
 			J9StackWalkState walkState = {0};
-			walkState.flags = J9_STACKWALK_INCLUDE_NATIVES | J9_STACKWALK_VISIBLE_ONLY | J9_STACKWALK_COUNT_SPECIFIED | J9_STACKWALK_RECORD_BYTECODE_PC_OFFSET;
-			walkState.skipCount = (UDATA) depth;
-			walkState.maxFrames = 1;
+			walkState.flags = J9_STACKWALK_INCLUDE_NATIVES | J9_STACKWALK_VISIBLE_ONLY | J9_STACKWALK_RECORD_BYTECODE_PC_OFFSET | J9_STACKWALK_ITERATE_FRAMES;
+			walkState.userData1 = (void *)(UDATA)depth;
```

Comment:
Though not strictly necessary, can you please explicitly zero the userData2 field here?